sympy/assumptions/facts.py

```diff
@@ -34,71 +36,138 @@ def get_composite_predicates():
 
 @cacheit
 def get_known_facts():
+    """
+    Facts between unary predicates.
+
+    Returns
+    =======
+
+    x : Placeholder symbol for unary facts.
+
+    fact : Known facts in conjugated normal form.
+
+    """
     # We build the facts starting with primitive predicates.
     # DO NOT include the predicates in get_composite_predicates()'s keys here!
-    return And(
-
+    x = Symbol('x')
```

Comment:
Instead of changing the return value, it would be simpler to add a default parameter to the function, like `get_known_facts(x=Symbol('x'))`.

Reply:
Or it might have to be

```py
def get_known_facts(x=None):
    if x is None:
        x = Symbol('x')
```

to avoid using Symbol at the module level.